### DISCUSSION TITLE
Tizen browser canvas blinking

### DISCUSSION THREAD

**Discussion by Faeren0327** [bevyengine/bevy#7816] [Q&A]
Hy!

I just realised that Bevy example (https://bevyengine.org/examples/3d/3d-scene/) is blinking when I browsing it using Tizen Web browser on a Samsung TV.
Is there a way fix that?

Thanks!

**Comment by Faeren0327** [comment]
I just tested with bevy_webgl2 plugin and it seems to work.
https://mrk.sed.pl/bevy-showcase/#contributors
Is there a way to solve it without using that plugin, because I have some package dependency issues.

**Reply by mockersf** [reply]
`bevy_webgl2` is deprecated and won't work with Bevy after the 0.5

**Comment by mockersf** [comment]
Do you have access to the console of that browser? Is there anything in the log?

**Reply by Faeren0327** [reply]
Unfortunately not. There is a simulator available (Tizen Web Simulator Application), but I could not make it work yet on my pc.
